packages/es-components/src/components/containers/tooltip/Tooltip.js

```diff
@@ -87,6 +87,7 @@ function Tooltip(props) {
     content,
     styleType,
     children,
+    buttonLabel,
```

Comment:
To go along with the suggested change regarding `linkProps` as an object.
```suggestion
    linkProps,
```